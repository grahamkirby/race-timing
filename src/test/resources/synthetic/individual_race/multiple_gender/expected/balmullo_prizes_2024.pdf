%PDF-1.7
%‚„œ”
5 0 obj
<</Filter/FlateDecode/Length 811>>stream
xú}ñÀV€0Ü˜~
ÌÄETçÆ∂w °¥Å“Æÿ"àã#«9úÚ∆}ã*Æ/RlwáŒôfÚy˛^£≥eÙ·Âh˘1âîíXJ¥\E«g:ﬂÏÚº@ÀRg9∫’èQ‚"Á∫2œE˘}-≥w≥=Y˛å>.£o—Îølm≥Q¿L’ŸZQän~ãÓåÕä“ì¨z´Eê‡X°M“ıí¥Ô<∫Îk∞æÜLfq]RtÿoyÇ˜cO7&œ4ZÏ≤ÚP”&P8,ÄÓèÔ™RWk£Kã.uYf¶‹ﬁü BRNSŒS	ØO≈0Ø≥–aõ2l≥˝¥’∆±œ™«uf¶≤ïXÅ's}^dOùŒõﬁx
‰P≠ºﬁ`Nk)6áÕı±W≈⁄}…MV≠ßrp,c_3hM§L Ø5"0Á#t]ÿŸYfµõ¡ˇí$'‹%d?HÕ;$ })°ÃÂ‘$º‡yÆÀtsÑÊÖµ≈`ú∫¿	éc_È`|∑∫™p√"N‰ûüs_x‘OWªIÇNZ?⁄æCû°Qò'ìBGy¡ßõ?h°œ´Û’@÷ôä”˝∑tH2åƒ%`c8ôÒdîó®∏?X˚HØX∞I*`‡_Í≠∂Ë∂x0eµ-Ï@(;
ÒûÇßtNÌ∂2•[+´Ufö≠"úsìAö∏Ôî≈	b⁄?åÀ©Ì(«òà⁄ ı¶kﬁ!œãL2,&BI@≈æ+û‹™Ω9öG[ò‹/zO8XI*‚A]:t+[ƒc≥!»Låœgã∏üçÊP†û	ix Ù±∑ÊŸM∆"À›≤àh∑»][ûhˇ˚u˛íkªjØŒ⁄Ó‰ï¬‡‰PÓµ ñtá$39éÉré]ÇGÛqxv§Ó<K>…#ºl^µq?gr_tÄ≈{$ΩŒ19ﬂŸï1ŒfzÂw‰RêÉTﬁ°w"Â°[j äÃ‘8PÉËÅ4ÔàÁH‡
K5	$	Äx¡_LÅèüç}^˙.?–á5zó˙ÌeÌÏ‡)]¬ø=ÜRC
endstream
endobj
4 0 obj
<</Contents 5 0 R/MediaBox[0 0 595 842]/Parent 2 0 R/Resources<</Font<</F1 6 0 R/F10 15 0 R/F11 16 0 R/F12 17 0 R/F13 18 0 R/F14 19 0 R/F15 20 0 R/F16 21 0 R/F17 22 0 R/F18 23 0 R/F19 24 0 R/F2 7 0 R/F20 25 0 R/F21 26 0 R/F22 27 0 R/F23 28 0 R/F24 29 0 R/F25 30 0 R/F26 31 0 R/F27 32 0 R/F28 33 0 R/F29 34 0 R/F3 8 0 R/F4 9 0 R/F5 10 0 R/F6 11 0 R/F7 12 0 R/F8 13 0 R/F9 14 0 R>>>>/TrimBox[0 0 595 842]/Type/Page>>
endobj
1 0 obj
<</Pages 2 0 R/Type/Catalog>>
endobj
3 0 obj
<</CreationDate(D:20250127195816Z)/ModDate(D:20250127195816Z)/Producer(iTextÆ Core 9.0.0 \(AGPL version\) ©2000-2024 Apryse Group NV)>>
endobj
6 0 obj
<</BaseFont/Helvetica/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
7 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
8 0 obj
<</BaseFont/Helvetica/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
9 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
10 0 obj
<</BaseFont/Helvetica/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
11 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
12 0 obj
<</BaseFont/Helvetica/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
13 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
14 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
15 0 obj
<</BaseFont/Helvetica/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
16 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
17 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
18 0 obj
<</BaseFont/Helvetica/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
19 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
20 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
21 0 obj
<</BaseFont/Helvetica/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
22 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
23 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
24 0 obj
<</BaseFont/Helvetica/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
25 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
26 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
27 0 obj
<</BaseFont/Helvetica/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
28 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
29 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
30 0 obj
<</BaseFont/Helvetica/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
31 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
32 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
33 0 obj
<</BaseFont/Helvetica/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
34 0 obj
<</BaseFont/Helvetica-Bold/Encoding/WinAnsiEncoding/Subtype/Type1/Type/Font>>
endobj
2 0 obj
<</Count 1/Kids[4 0 R]/Type/Pages>>
endobj
xref
0 35
0000000000 65535 f 
0000001323 00000 n 
0000004186 00000 n 
0000001368 00000 n 
0000000893 00000 n 
0000000015 00000 n 
0000001519 00000 n 
0000001607 00000 n 
0000001700 00000 n 
0000001788 00000 n 
0000001881 00000 n 
0000001970 00000 n 
0000002064 00000 n 
0000002153 00000 n 
0000002247 00000 n 
0000002341 00000 n 
0000002430 00000 n 
0000002524 00000 n 
0000002618 00000 n 
0000002707 00000 n 
0000002801 00000 n 
0000002895 00000 n 
0000002984 00000 n 
0000003078 00000 n 
0000003172 00000 n 
0000003261 00000 n 
0000003355 00000 n 
0000003449 00000 n 
0000003538 00000 n 
0000003632 00000 n 
0000003726 00000 n 
0000003815 00000 n 
0000003909 00000 n 
0000004003 00000 n 
0000004092 00000 n 
trailer
<</ID [<9f1ba963ae2b3e405fb2557d0f007aa2ab9f081439b1d61c9337950a3d3cbac457bbb5c4d8b37733eea714cfa339de85c7f8f50df268a7b8aa4736bca6e05793><9f1ba963ae2b3e405fb2557d0f007aa2ab9f081439b1d61c9337950a3d3cbac457bbb5c4d8b37733eea714cfa339de85c7f8f50df268a7b8aa4736bca6e05793>]/Info 3 0 R/Root 1 0 R/Size 35>>
%iText-Core-9.0.0
startxref
4237
%%EOF
