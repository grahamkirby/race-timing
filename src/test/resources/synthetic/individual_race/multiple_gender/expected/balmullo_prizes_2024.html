<h4>Prizes</h4>
<p><strong>Open Senior</strong></p>
<ul>
    <li>1 Amelia Muir (Strathearn Harriers) 00:42:43</li>
    <li>2 Isabel Ritchie (Fife AC) 00:44:10</li>
    <li>3 John Smith (Fife AC) 00:45:37</li>
</ul>

<p><strong>Non-Binary Senior</strong></p>
<ul>
    <li>1 Clark O' Connor (Unatt.) 00:48:31</li>
</ul>

<p><strong>Open Junior</strong></p>
<ul>
    <li>1 Am&eacute; MacDonald (Unatt.) 01:00:07</li>
</ul>

<p><strong>Open 40-49</strong></p>
<ul>
    <li>1 Hasan Robertson (Anster Haddies) 00:54:19</li>
</ul>

<p><strong>Non-Binary Veteran</strong></p>
<ul>
    <li>1 Sofia O'Connor (Fife AC) 00:49:58</li>
</ul>

<p><strong>Open 50-59</strong></p>
<ul>
    <li>1 Regan Millar (Falkland Trail Runners) 00:57:13</li>
</ul>

<p><strong>Open 60-69</strong></p>
<ul>
    <li>1 Neil MacDonald (Dundee Road Runners) 01:10:16</li>
</ul>

<p><strong>Open 70-79</strong></p>
<ul>
    <li>1 Leo McKenzie (Dundee Hawkhill Harriers) 00:51:25</li>
</ul>

